### PR TITLE
Avoid poll(None) if no timeout specified

### PR DESCRIPTION
**Pull request by dwalton76**
Potential fix for #583

### CODE REVIEW

File: ev3dev2/motor.py [hunk 1]
```diff
@@ -865,14 +865,22 @@ def wait(self, cond, timeout=None):
             self._poll = select.poll()
             self._poll.register(self._state, select.POLLPRI)
 
+        # Set poll timeout to something small
```
Comment (WasabiFan) [inline ev3dev2/motor.py]: Can you add a comment with a link to the original issue for future reference?

**Review by WasabiFan — APPROVED** [review]
I'd like to see verification that it solves the problem, but this looks good to me in theory.

**Comment by dwalton76** [comment]
I think I had worked around this problem in my cubesolver by putting multi second limits just about everywhere I use wait...so I don't have a good way to test/verify that this fixes the issue. I would be fine if we go ahead and commit it though, the fix looks pretty straightforward and should fix the issue.

**Comment by WasabiFan** [comment]
As long as it doesn't make behavior worse, I'm OK with merging.

**Comment by dwalton76** [comment]
I did some basic testing with python3 and micropython and it seems fine...will merge.

**Pull request merged**